vendor.conf

```diff
@@ -23,6 +23,13 @@ github.com/docker/libnetwork 3ab699ea36573d98f481d233c30c742ade737565
 github.com/opencontainers/runc 8e8d01d38d7b4fb0a35bf89b72bc3e18c98882d7
 github.com/opencontainers/go-digest a6d0ee40d4207ea02364bd3b9e8e77b9159ba1eb
 
+github.com/docker/containerd 1dc5d652ac1adf8f0a92ee8eff7af07b129d4e21
+github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
+github.com/gorilla/context v1.1
```

Comment:
They seem to be brought in as a side effect of importing `distribution` packages to parse manifests. I'd prefer if we could move manifest-related functionality out of swarmkit.